k8s/staging/ingress-controller-patch.json

```diff
@@ -0,0 +1,12 @@
+[
```

Comment:
this patch specific to the `staging` env adds [two more args to the `nginx` controller](https://github.com/tablelandnetwork/basin-provider/pull/11/files#diff-277d9645bd55e6669440b63397a4efcfa0f469e0d07ec41d9c89e34f1c79b866R401-R410):
- `tcp-services-configmap`: tells nginx how to deal with TCP connections. Take a look at the [`configTcpServices.env`](https://github.com/tablelandnetwork/basin-provider/pull/11/files#diff-fa9bc908c45122f324a584d1bf57d4d4d2084070a79554041e983556f31fd3b3R1). This config means, any TCP connection to port 3000, will be proxied to [`basin-staging/basin-worker` k8s Service](https://github.com/tablelandnetwork/basin-provider/pull/11/files#diff-7d80e8598579da72912720feac45bae9b6f4581e5ee0aaa167d5703879b12176R1-R6) at port 3000.